ocaml/idl/schematest.ml

```diff
@@ -3,7 +3,7 @@ let hash x = Digest.string x |> Digest.to_hex
 (* BEWARE: if this changes, check that schema has been bumped accordingly in
    ocaml/idl/datamodel_common.ml, usually schema_minor_vsn *)
 
-let last_known_schema_hash = "96da2c136aa13d4f8e71dd40fe2b84af"
```

Comment:
Please change `schema_minor_vsn` in `ocaml/idl/datamodel_common.ml`, as the comment above instructs